packages/restapi/tests/lib/notification/alias.test.ts

```diff
@@ -20,34 +26,141 @@ describe('PushAPI.alias functionality', () => {
       : new (ethers as any).JsonRpcProvider('https://rpc.sepolia.org');
 
+    const provider1 = (ethers as any).providers
+      ? new (ethers as any).providers.JsonRpcProvider('https://bsc-testnet-rpc.publicnode.com')
+      : new (ethers as any).JsonRpcProvider('https://bsc-testnet-rpc.publicnode.com');
+
+
     signer2 = new ethers.Wallet(
       `0x${process.env['WALLET_PRIVATE_KEY']}`,
       provider
     );
+
+    const signer3 = createWalletClient({
+      account: privateKeyToAccount(`0x${process.env['WALLET_PRIVATE_KEY']}`),
+      chain: sepolia,
+      transport: http(),
+    });
+
+
+    const signer4 = new ethers.Wallet(
+      `0x${process.env['NFT_HOLDER_WALLET_PRIVATE_KEY_1']}`,
+      provider1
+    );
+
+    const signer5 = createWalletClient({
+      account: privateKeyToAccount(`0x${process.env['NFT_HOLDER_WALLET_PRIVATE_KEY_1']}`),
+      chain: bscTestnet,
+      transport: http(),
+    });
+
     account2 = await signer2.getAddress();
 
-    // initialisation with signer and provider
+    // initialization with signer
+    userAlice = await PushAPI.initialize(signer1, { env: _env });
+
+    // initialization with signer and provider
     userKate = await PushAPI.initialize(signer2, { env: _env });
-    // initialisation with signer
-    userAlice = await PushAPI.initialize(signer2, { env: _env });
-    // TODO: remove signer1 after chat makes signer as optional
+    
     //initialisation without signer
     userBob = await PushAPI.initialize(signer1, { env: _env });
+
+    // initalisation with viem
+    viemUser = await PushAPI.initialize(signer3, { env: _env });
+
+    // initalisation the verification account
+    userSam = await PushAPI.initialize(signer4, { env: _env });
+    viemUser1 = await PushAPI.initialize(signer5, { env: _env });
+
   });
 
   describe('alias :: info', () => {
-    // TODO: remove skip after signer becomes optional
     it('Should return response', async () => {
       const res = await userBob.channel.alias.info({
         alias: '0x93A829d16DE51745Db0530A0F8E8A9B8CA5370E5',
         aliasChain: 'POLYGON',
       });
-      //   console.log(res)
       expect(res).not.null;
     });
   });
+
+  describe('alias :: add', () => {
+    // TODO: remove skip after signer becomes optional
+    it('Without signer and account :: should throw error', async () => {
+      await expect(() =>
+        userBob.channel.alias.initiate(
+          'eip155:11155111:0x74415Bc4C4Bf4Baecc2DD372426F0a1D016Fa924'
+        )
+      ).to.Throw;
+    });
+
+    it('With signer and without provider :: should throw error', async () => {
+      await expect(() =>
+        userAlice.channel.alias.initiate(
+          'eip155:11155111:0x74415Bc4C4Bf4Baecc2DD372426F0a1D016Fa924'
+        )
+      ).to.Throw;
+    });
+
+    it('With signer and provider :: should add alias', async () => {
+      const res = await userKate.channel.alias.initiate(
```

Comment:
I don't think the test cases are correct
According to the approach
1. To initiate correctly userKate should call ETH Comm contract ( currently its calling BSC Comm Contract )
2. After that we should check if push nodes now has a unerified alias for userKateChannel
3. Then for alias verification, BSC Comm contract will be called